README.md

```diff
@@ -563,3 +563,25 @@ To access the data, please use the [Azure Portal](https://portal.azure.com).
 ### Source Code Structure
 
 Where appropriate the source code must be split out into a hierarchy by appeal type. Several examples of this may be found under cypress tests (i.e. /packages/e2e-tests/cypress/integration), forms-web-app (i.e. /packages/forms-web-app/src/routes/), and many others.
+
+## Feature Flag
+
+We have incorporated Azure feature flag functionality into the common package for use across the solution. Feature flags should be set up in the infra, and can be used by importing the `isFeatureActive` function from `@pins/common`, i.e; 
```

Comment:
Just a minor point, could this be explicit about what's meant by "in the infra"? Where specifically are the feature flags set up?